tests/functional/parse.t

```diff
@@ -31,4 +31,22 @@ Parse compressed Zika sequences into sequences and metadata.
   $ diff -u "parse/metadata.tsv" "$TMP/metadata.tsv"
   $ rm -f "$TMP/sequences.fasta" "$TMP/metadata.tsv"
 
+Error on the first duplicate.
+
+  $ echo -e '> SEQ1' >> $TMP/data.fasta
+  $ echo -e 'AAA'    >> $TMP/data.fasta
+  $ echo -e '> SEQ1' >> $TMP/data.fasta
+  $ echo -e 'AAA'    >> $TMP/data.fasta
+  $ echo -e '> SEQ2' >> $TMP/data.fasta
+  $ echo -e 'AAA'    >> $TMP/data.fasta
+  $ echo -e '> SEQ2' >> $TMP/data.fasta
+  $ echo -e 'AAA'    >> $TMP/data.fasta
```

Comment:
The `-e` is unnecessary here. I also think using a heredoc (example below) to be a bit less wordy, but YMMV. Take it or leave it!

```suggestion
  $ cat >$TMP/data.fasta <<~~
  > >SEQ1
  > AAA
  > >SEQ1
  > AAA
  > >SEQ2
  > AAA
  > >SEQ2
  > AAA
  > ~~
```